projects/ad2s1210_iio/README.txt

```diff
@@ -11,7 +11,7 @@ Use of the Mbed platform allows code to be ported to other Mbed supported target
 
 Product details: AD2S1210.
 Product Evaluation board details: EVAL-AD2S1210SDZ
-User Guide for this code: https://wiki.analog.com/resources/tools-software/product-support-software/AD2S1210_mbed_iio_support
+User Guide for this code: https://analogdevicesinc.github.io/precision-converters-firmware/source/projects/ad2s1210_iio/ad2s1210_iio.html
```

Comment:
do we really need the readme?

Reply:
Most of the projects added before GH docs have the readme files. Discussed the same with @gprasadbng, and decided to leave it as is for older projects fixing the broken links.

Reply:
In that case, should we update all the wiki links in this readme?
There is another link that points to wiki in line-54 "Find detailed instructions here...."